sky/check.py

```diff
@@ -61,4 +62,12 @@ def get_cloud_credential_file_mounts() -> Dict[str, str]:
     for cloud in enabled_clouds:
         cloud_file_mounts = cloud.get_credential_file_mounts()
         file_mounts.update(cloud_file_mounts)
+    # Currently, get_enabled_clouds() does not support r2
+    # as only clouds with computing instances are supported
+    # by 'clouds'
+    if cloudflare.r2_is_enabled():
+        if '~/.aws/credentials' not in file_mounts:
```

Comment:
Instead of hardcoding files here, can we have something similar to what we have for enabled clouds? Something like:
```
r2_credential_mounts = cloudflare.get_credential_file_mounts()
file_mounts.update(r2_credential_mounts)
```